src/cf-typography/src/atoms/headings.less

```diff
@@ -0,0 +1,31 @@
+.a-heading {
+
+    &__padded {
```

Comment:
Why nest the modifiers in this file but not others? I'd go in the direction of un-nesting here, especially since `.a-heading` has no styles of its own.